plugins/arkworks/src/constraint.rs

```diff
@@ -124,9 +124,8 @@ where
             BITS < F::Params::MODULUS_BITS as usize,
             "BITS must be strictly less than modulus bits of `F`."
         );
-        let value_bits = value
-            .to_bits_le()
-            .expect("Bit decomposition is not allowed to fail.");
+        let value_bits =
+            ToBitsGadget::to_bits_le(value).expect("Bit decomposition is not allowed to fail.");
```

Comment:
Should we use bit decomposition trait here and use this to build a generic impl of `assert_within_range`. If you think it's a good idea mark an issue for it.

Reply:
https://github.com/openzklib/openzl/issues/55